kubernetes/chart/zulip/values.yaml

```diff
@@ -177,6 +177,17 @@ postSetup:
     #   #!/bin/bash
     #   echo "This is a script that gets executed in the Zulip container after installation, once migrations are complete"
 
+  # -- You can add any sidecar to Zulip, like a minio client to use for
+  # uploading backups to a bucket, and mounting the volume for Zulip.
+  # sidecars:
```

Comment:
This isn't new in this PR, but this comment style isn't very readable; should we think about switching to using the double-# pattern for the text part/
```
## text comment
# commented code
```

If you like the idea, just put up a follow-up PR for it.

Reply:
The commenting format is for [helm docs](https://github.com/norwoodj/helm-docs) - and as you have README.md.gotmpl that uses helm docs for the README - I think it should be good to keep it. Maybe I also should have added the change to the README generated by Helm docs as part of pull request?